### PR TITLE
Register prathamesh.is-a.dev

### PR DESCRIPTION
**Pull request by prathamesh-dukare**
Register prathamesh.is-a.dev with CNAME record pointing to prathamesh.works.

### CODE REVIEW

**Review by iostpa — APPROVED** [review]
seems good to me

**Comment by CuteDog5695** [comment]
you have to wait for one person in trial-maintainers to approve this PR and merge it since we changed code owners (which seems to screw it up for us maintainers)

**Pull request merged**
